crates/hotshot/src/tasks/mod.rs

```diff
@@ -252,7 +252,7 @@ where
 /// # Panics
 /// Is unable to panic. This section here is just to satisfy clippy
 pub async fn add_consensus_task<
-    TYPES: NodeType<BlockType = VIDBlockPayload>,
+    TYPES: NodeType<BlockHeader = VIDBlockHeader, BlockPayload = VIDBlockPayload>,
```

Comment:
Why do we fix `BlockHeader` here? Shouldn't the application be able to set some header type other than `VIDBlockHeader`?